deploy/deploy.sh

```diff
@@ -29,6 +33,15 @@ else
   exit 1
 fi
 
+CURRENT_BONFIRE_VERSION=$(bonfire version | cut -d' ' -f3)
+
+if [[ $(printf '%s\n' "$MIN_BONFIRE_VERSION" "$CURRENT_BONFIRE_VERSION" | sort -V | head -n1) != "$MIN_BONFIRE_VERSION" ]]; then
+  echo "Current bonfire version ($CURRENT_BONFIRE_VERSION) is less than required version ($MIN_BONFIRE_VERSION)."
+  echo "Please upgrade bonfire with command:"
+  echo "pip install crc-bonfire"
```

Comment:
i assume it should be `pip install --upgrade crc-bonfire` to upgrade?

Reply:
@Rajagopalan-Ranganathan absolutely, good eye, thanks! (I updated it)